pkg/helpers/queuekey_test.go

```diff
@@ -204,7 +204,7 @@ func TestClusterManagerDeploymentQueueKeyFunc(t *testing.T) {
 			fakeOperatorClient := fakeoperatorclient.NewSimpleClientset(c.clusterManager)
 			operatorInformers := operatorinformers.NewSharedInformerFactory(fakeOperatorClient, 5*time.Minute)
 			store := operatorInformers.Operator().V1().ClusterManagers().Informer().GetStore()
-			store.Add(c.clusterManager)
+			_ = store.Add(c.clusterManager)
```

Comment:
I think it is better to add error check here like

```go
if err := store.Add(c.clusterManager); err != nil {
    t.Fatal(err)
}
```

Reply:
Solved.